app/sidekiq/form1010cg/submission_job.rb

```diff
@@ -21,10 +21,8 @@ class SubmissionJob
       StatsD.increment("#{STATSD_KEY_PREFIX}failed_no_retries_left", tags: ["claim_id:#{msg['args'][0]}"])
       StatsD.increment('silent_failure_avoided_no_confirmation', tags: DD_ZSF_TAGS)
 
-      if Flipper.enabled?(:caregiver_use_va_notify_on_submission_failure)
-        claim = SavedClaim::CaregiversAssistanceClaim.find(msg['args'][0])
-        send_failure_email(claim.parsed_form) if claim.parsed_form.dig('veteran', 'email')
-      end
+      claim = SavedClaim::CaregiversAssistanceClaim.find(msg['args'][0])
```

Comment:
It could, although realistically it shouldn't. I'm not sure if doing a `find_by` helps us a ton here, though? If the record is not found we won't have an email to send to, so throwing an exception here would just fail. If it's gotten to this block it has already run out of retries, so it would just not send the email. I guess we could do a `find_by` and then log that we couldn't find the saved claim to send an email if it returns false?